packages/vitest/src/node/types/browser.ts

```diff
@@ -236,6 +236,21 @@ export interface BrowserConfigOptions {
    * @default 30000
    */
   connectTimeout?: number
+
+  expect?: {
+    toMatchScreenshot?: {
+      [ComparatorName in keyof ToMatchScreenshotComparators]:
+      {
+        /**
+         * The name of the comparator to use for visual diffing.
+         *
+         * @defaultValue `'pixelmatch'`
+         */
+        comparatorName?: ComparatorName
+        comparatorOptions?: ToMatchScreenshotComparators[ComparatorName]
+      }
+    }[keyof ToMatchScreenshotComparators] & ToMatchScreenshotOptions
```

Comment:
This type is a bit chaotic, but without generics the only solution is to create discriminated unions.

The interface is compatible with `ScreenshotComparatorRegistry`, so I used that to extend it in the providers. In the future this will allow to add more comparators and have them typed correctly in the config by augmenting `ScreenshotComparatorRegistry`.